core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -175,13 +188,20 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     this.languageCode = this.activeSuggestion.change.
       language_code;
     this.userService.getUserInfoAsync().then(userInfo => {
-      this.username = userInfo.getUsername();
+      const username = userInfo.getUsername();
+
+      if (username === null) {
+        throw new Error('Username does not exist.');
+      }
+      this.username = username,
```

Comment:
Why comma instead of semicolon, at the end?

Reply:
Done